src/cpp-mariadb/devcontainer-template.json

```diff
@@ -14,6 +14,7 @@
 				"debian-12",
                 "debian-11",
                 "debian-10",
+				"ubuntu-24.04",
```

Comment:
Can we update the description for this Template as well?

Reply:
done